TicTacToe.cpp

```diff
@@ -1,6 +1,22 @@
 #include "TicTacToe.h"
+#include <time.h>
```

Comment:
I think you also need `#include <stdlib.h>` ([cit](http://www.cplusplus.com/reference/cstdlib/rand/))

Reply:
Ah, it's fine. No compiler errors, so this is al good. Nevermind my previous comment 😄